main.go

```diff
@@ -93,6 +95,11 @@ func StartCmd() cli.Command {
 				Usage: "Required. Specify configmap name.",
 				Value: DefaultConfigMapName,
 			},
+			cli.StringFlag{
+				Name:   FlagPullSecret,
+				Usage:  "The pullsecret for helper images",
```

Comment:
`ImagePullSecret for the helper image, if needed`

Reply:
Alernatively, we could write the `imagePullSecret` into the ServiceAccount in the chart, and use it in every pods, including the helper pods.

Reply:
Good feedback. On the question on binding imagepullsecret to SA I feel this is an architecture design. In our setup this would work nicely. I´ll play around with the SA binding and either amend tmpl changes or new design (if sa binding is preferred)

Reply:
Yeah, adding it to the service account is a good idea. Then we don't need to change the code of the provisioner. But we should document that or adding an option in the chart for that.